website/docs/intro/install/alpine.mdx

```diff
@@ -5,17 +5,20 @@ description: |-
   Install OpenTofu on Alpine Linux.
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import ApkConvenienceScript from '!!raw-loader!./alpine-convenience.sh'
+import ApkManualScript from '!!raw-loader!./alpine-manual.sh'
+
 # Installing OpenTofu on Alpine Linux
 
 OpenTofu is available in the [Alpine Linux testing repository](https://pkgs.alpinelinux.org/packages?name=opentofu) or as an `.apk` package from the [GitHub releases page](https://github.com/opentofu/opentofu/releases/latest/).
 
-## Installing from the testing repository
+## Installing using the installer
 
-OpenTofu is available in the Alpine Testing repository and can be installed from there:
+You can use the OpenTofu installer script to run the installation.
 
-```bash
-apk add opentofu --repository=https://dl-cdn.alpinelinux.org/alpine/edge/testing/
-```
+<CodeBlock language="bash">{ApkConvenienceScript}</CodeBlock>
```

Comment:
This `CodeBlock` and the one below don't render as actual code blocks like the previous code did and the apk section are
![image](https://github.com/opentofu/opentofu/assets/62661/65bfedd8-d2cd-4593-99a5-0beede505aea)

Reply:
Thank you for catching this! It seems I forgot an import.